Floofbot/Migrations/FloofDataContextModelSnapshot.cs

```diff
@@ -18,8 +18,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             modelBuilder.Entity("Floofbot.Services.Repository.Models.LogConfig", b =>
                 {
+                    b.Property<ulong>("ServerId")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("INTEGER");
+
                     b.Property<bool>("IsOn")
-                        .HasColumnType("INT");
+                        .HasColumnType("INTEGER");
```

Comment:
why is it modifying the logconfig model?

Reply:
It's recognizing which fields are primary keys, which explains the HasKey calls.
I'm not sure why it's changing the type from Int to Integer, though.